Resources/Prototypes/DeltaV/Roles/Jobs/Fun/misc_startinggear.yml

```diff
@@ -63,4 +63,23 @@
     gloves: ClothingHandsGlovesCombat
     shoes: ClothingShoesSlippers
     id: SyndiPDA
-  innerClothingSkirt: ClothingUniformJumpsuitPyjamaSyndicatePink
\ No newline at end of file
+  innerClothingSkirt: ClothingUniformJumpsuitPyjamaSyndicatePink
+  
+# Syndicate Paramedic
+
+- type: startingGear
+  id: SyndicateParamedicGear
+  equipment:
+    jumpsuit: UniformScrubsColorCybersun
+    back: ClothingBackpackDuffelSyndicateFilledMedical
+    shoes: ClothingShoesChameleonNoSlips
+    ears: ClothingHeadsetAltSyndicate
+    eyes: ClothingEyesHudSyndicateMedical
+    head: ClothingHeadHatSurgcapCybersun
+    outerClothing: ClothingOuterCoatCybersunWindbreaker
+    gloves: ClothingHandsGlovesCombat
+    id: SyndiPDA
+    belt: ClothingBeltParamedicFilled
+    pocket1: HandheldGPSBasic
+    pocket2: HandheldCrewMonitor
```

Comment:
I feel like this is an extremely broken item to be available for the low low price of 10 tc